packages/contentstack-import/src/import/modules/base-class.ts

```diff
@@ -381,6 +384,19 @@ export default abstract class BaseClass {
           .delete({ locale: additionalInfo.locale })
           .then(onSuccess)
           .catch(onReject);
+      case 'create-taxonomies':
+        return new HttpClient()
+          .headers(additionalInfo.headers)
+          .post(additionalInfo.url, { taxonomy: apiData })
+          .then(onSuccess)
+          .catch(onReject);
+      case 'create-terms':
+        const url = `${additionalInfo.baseUrl}/${apiData.taxonomy_uid}/terms`;
+        return new HttpClient()
+          .headers(additionalInfo.headers)
+          .post(url, { term: apiData })
+          .then(onSuccess)
+          .catch(onReject);
```

Comment:
Can this `HttpClient` instance be created globally instead of being created every time the case is triggered, considering its usage in multiple locations?